pkgs/data/sgml+xml/schemas/sgml-dtd/docbook/4.2.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+let
+
+  src = fetchurl {
+    url = "http://www.oasis-open.org/docbook/sgml/4.2/docbook-4.2.zip";
+    sha256 = "Z+vSyUs0JxjGhl0t5g9dT/Atd6fksNnnKkjEXysmNcM=";
```

Comment:
```suggestion
    sha256 = "sha256-Z+vSyUs0JxjGhl0t5g9dT/Atd6fksNnnKkjEXysmNcM=";
```
or
```suggestion
    hash = "sha256-Z+vSyUs0JxjGhl0t5g9dT/Atd6fksNnnKkjEXysmNcM=";
```

Reply:
Do not see benefit in duplicating the hash type in legacy attribute and do not want to use the new one until there is consensus.

Reply:
We can only strip ``sha256:`` not ``sha256-`` because that is an important part for the base64 encoded hash and also makes the migration path to hash a lot easier when you can just change the variabel name.

Reply:
I disagree. The hash type prefix is important part of the SRI hash but the base64-encoded hash makes sense stand-alone as well, just like base16 or base32 – as long as the hash type is provided out of band in the attribute name.

There are already various different hash formats including this one in use so just a simple attribute replacement is not an option. And this format is just another simple branch in an `if` statement.

Reply:
We probably have other occurrences of this by now...